index.js

```diff
@@ -1,19 +1,21 @@
 'use strict';
 const path = require('path');
-const pathExists = require('path-exists');
+const locatePath = require('locate-path');
+
+const findPath = (dir, files) => locatePath(files.map(file => path.join(dir, file)), {concurrency: 1});
 
 module.exports = (filename, opts) => {
 	opts = opts || {};
 
 	const startDir = path.resolve(opts.cwd || '');
 	const root = path.parse(startDir).root;
 
+	const filenames = [].concat(filename);
+
 	return new Promise(resolve => {
 		(function find(dir) {
-			const fp = path.join(dir, filename);
-
-			pathExists(fp).then(exists => {
-				if (exists) {
+			findPath(dir, filenames).then(fp => {
```

Comment:
Before I add the post-find path joining logic myself here, is it by design that `locate-path` always only return the filename?